hm10_1.py

```diff
@@ -0,0 +1,37 @@
+""" Task 1: function arguments are positive numbers """
+
+
+def decorator(_any):
+
+    """ Function takes arguments _any """
+
+    def wrapper(*args):
+
+        """ Function takes arguments in any value """
+
+        lst = []
+        for x in args:
+            lst.append(x)
+        for i, y in enumerate(lst, start=1):
+            if isinstance(y, int):
+                if y > 0:
+                    print(f"{i}. {y} Число больше 0")
+                elif y == 0:
+                    raise ValueError(f"{i}. {y} Число равно 0")
+                elif y < 0:
+                    raise ValueError(f"{i}. {y} Число меньше 0")
+            else:
+                raise ValueError(f"{i}. {y} Невозможно сравнить")
+                # i += 1
```

Comment:
Positive value, value less the 0

```suggestion
                if y < 0:
                    raise ValueError(f"{i}. {y} Невозможно сравнить")
```

Reply:
Deleted exrta code and used args, ok
Do not use your advice:
1. it is easier to understand the numeric of the value
2. we need to understand what we have: number or not. And only then compare

Reply:
Regarding `isinstance` agreed, but what about to change `if-elif-elif` to just 1 `if`?

Reply:
Mmm, the task is to lavidate the argument: is it positive. If not positive - tell it. 0 - not positive, not negative. If make "Positive value, value less the 0", what about 0 and not numeric/float.
Simple is better than complex, logical is better than confusing :) For me my logic is more logical :)